jenkins-integration/jenkins-jobs/common/scripts/jenkins/pipeline.groovy

```diff
@@ -424,7 +424,8 @@ def single_pipeline(job) {
         if (job_name == "puppetserver-infinite") {
 
             dir('simulation-runner/config/scenarios') {
-                job["gatling_simulation_config"] = generate_gatling_scenario(NUMBER_OF_HOURS, CATALOG_SIZE, NODE_COUNT)
+                scenario_name = generate_gatling_scenario(NUMBER_OF_HOURS, CATALOG_SIZE, NODE_COUNT)
+                job["gatling_simulation_config"] = "../simulation-runner/config/scenarios/${scenario_name}"
```

Comment:
The relative path is fine here? We won't ever need to reference this from some other location than we are doing currently?

Reply:
The job hash drives everything else as far as i know. I have a for real job based on this branch running here as proof: http://puppetserver-perf-driver55.delivery.puppetlabs.net:8080/job/puppetserver-infinite/18/console

Reply:
Or I guess I should probably say, if we ever need to reference it from somewhere else we'll be making a bunch of other changes already, as it is hardcoded in all of the Jenkinsfiles and seems to be an assumption of everything.